CMake/kwiver_config.h.in

```diff
@@ -0,0 +1,17 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
```

Comment:
I think according to our latest style guide, we not want to avoid `/**` style doxygen documentation and use only `///` style instead.